Automattic/wp-calypso
# Users already on Pro are being asked to upgrade to Pro for two Earn features

#### Changes proposed in this Pull Request

This adds the two Earn Features to the Pro plan config and updates the related card copies to mention Pro plan.

#### Testing instructions

* Go to `/earn/[site]?flags=plans/pro-plan` on a Free site
* The "Collect PayPal payments" and "Earn ad revenue" should show the `Unlock this Feature` buttons .
<img width="320" alt="Screenshot on 2022-03-30 at 11-55-09" src="https://user-images.githubusercontent.com/2749938/160792798-36611305-9846-47ff-97d9-4a0d2399133d.png">


* Go to `/earn/[site]?flags=plans/pro-plan` on a Pro site
* The "Collect PayPal payments" and "Earn ad revenue" should be unlocked
<img width="320" alt="Screenshot on 2022-03-30 at 12-01-22" src="https://user-images.githubusercontent.com/2749938/160793985-1be286ab-43d1-470a-9a60-a16aab69fb1e.png">
